airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/common.py

```diff
@@ -345,31 +310,3 @@ async def _run(self, *args: Any, **kwargs: Any) -> StepResult:
                 stdout=f"Connector is certified but is still using a Dockerfile. {migration_hint}",
             )
         return StepResult(step=self, status=StepStatus.SUCCESS, stdout="Connector is certified and uses our base image.")
-
-
-class CheckPythonRegistryPublishConfiguration(Step):
```

Comment:
This is now checked inside the `connectors_qa` package https://github.com/airbytehq/airbyte/pull/35322